apps/drec-ui/src/device/containers/smartMeter/SmartMeterBlock/SmartMeterBlock.tsx

```diff
@@ -0,0 +1,22 @@
+import { Grid, Paper } from '@material-ui/core';
+import React, { FC } from 'react';
+import { SmartMeterChart } from '../SmartMeterChart';
+import { useStyles } from './SmartMeterBlock.styles';
+import { DeviceDTO } from '@energyweb/origin-drec-api-client';
+
+interface SmartMeterBlockProps {
+    device: DeviceDTO;
+}
+
+export const SmartMeterBlock: FC<SmartMeterBlockProps> = ({ device }) => {
+    const classes = useStyles();
+    return (
+        <Paper className={classes.paper}>
+            <Grid container spacing={4}>
+                <Grid item xs={12} md={12}>
```

Comment:
You don't need this Grid.item placement if you are having a single item which is always fullWidth

Reply:
Actually I had to add a class with width: 100% otherwise it wasn`t full width